TeamMemberFiles/aaron.voymas.txt

```diff
@@ -0,0 +1,8 @@
+Aaron Voymas
+Call me Aaron
+I am a COMP 294 student (394 next term and 495 next summer)
+My preferred tech stack is .Net/C#
+I look forward to learning and growing as a software engineer
+
+This line is for Activity 3 merge conflict
+This line is to cause a conflict for Activity 3

```

Comment:
Great job! Please make sure to clean up your personal repository to avoid exposing any information to the public that you wish to keep private.